forge-gui/res/cardsfolder/upcoming/andrios_roaming_explorer.txt

```diff
@@ -0,0 +1,11 @@
+Name:Andrios, Roaming Explorer
+ManaCost:5 G
+Types:Legendary Artifiact Creature Wizard
+PT:4/3
+K:Reach
+S:Mode$ Continuous | CharacteristicDefining$ True | SetPower$ 16 | SetToughness$ 9 | IsPresent$ Card.Self+attacking | Affected$ Creature.basePowerEQ4+baseToughnessEQ3+tapped+YouCtrl | Description$ As long as NICKNAME is attacking, tapped creatures you control with base power and toughness 4/3 have base power and toughness 16/9. 
+A:AB$ Mana | Cost$ T | Produced$ W U B R G | TriggersWhenSpent$ TrigChange | SpellDescription$ Add {W}{U}{B}{R}{G}. Creature spells you spend this mana to cast have their base power and toughness become 4/3.
+SVar:TrigChange:Mode$ SpellCast | ValidCard$ Creature | ValidActivatingPlayer$ You | Execute$ TrigEffect | TriggerDescription$ Creature spells you spend this mana to cast have their base power and toughness become 4/3.
+SVar:TrigEffect:DB$ Effect | RememberObjects$ TriggeredCard | StaticAbilities$ ChangeBase | Duration$ Permanent
+SVar:ChangeBase:Mode$ Continuous | Affected$ Card.IsRemembered | SetPower$ 4 | SetToughness$ 3 | Description$ This card has base power and toughness 4/3
```

Comment:
Description$ This creature has base power and toughness 4/3.

Maybe reads better?